### ISSUE TITLE
Translation: Irish (Gaeilge)

### ISSUE THREAD

**Issue by RobertDev** [chubin/wttr.in#305]
It would be great to have wttr translated into Irish.
I'm happy to work on the translation, over the next week or two!

**Comment by chubin** [comment]
@RobertDev Robert thank you very much for this!
We are missing the Irish translation very badly, and it would be really great if we could add it.

First of all, could you please translate these 5 words/phrases:

1. `Weather report for:`
2. `Morning`
3. `Day`
4. `Evening`
5. `Night`

After that you can fork this repository and I will say to you, what files and where should be added, extended etc.

**Comment by RobertDev** [comment]
@chubin Happy to help out! 

Sure:
1. `Réamhaisnéis na haimsire do: <PlaceName>`

2. `Maidin`

3. ` Lá ` that literally means the noun for a day. Depending on the meaning you're trying to convey you might prefer Noon: `Nóin`, Or Daytime: `I rith an lae`

4. `Tráthnóna`

5. `Oíche`

**Comment by chubin** [comment]
@RobertDev Robert, thank you very much for your translation. I've added initial support of Irish:

```
$ curl ga.wttr.in
```

That's how it looks like:

![_lang=ga](https://user-images.githubusercontent.com/3875145/58652397-c4bf4f00-8313-11e9-9fcf-2abb15fbcf24.png)

Now we need the following things:

1. Translations for weather conditions  You can take say `cy.txt` as a base for your file.
```
cp share/translations/cy.txt share/translation/ga.txt
vim share/translation/ga.txt
```
2. Translations of various messages in `lib/translations.py`

That is the main part. After it is done, to finalize the translation, we need the last step, translation of help:

3. Translation of the help.
```
cp share/translations/help.txt share/translations/ga-help.txt
vim share/translations/ga-help.txt
```

After each step you can send me a pull request to switch the newly translated part live.

**Comment by chubin** [comment]
@RobertDev Robert, sorry for disturbing you, I just wanted to ask, may I somehow help with the trnanslation? It would be really great to have Irish support on wttr.in

**Comment by ivernus** [comment]
> @RobertDev Robert, sorry for disturbing you, I just wanted to ask, may I somehow help with the trnanslation? It would be really great to have Irish support on wttr.in

Hopefully the updated translations in the commit look like they are in the correct places?

**Comment by chubin** [comment]
@ivernus yes, it looks very good. A couple of files are missing (e.g. help.txt translation),
but it is perfect finally have the Irish translation even without the help file.
Could you please make a pull-request and I will merge and deploy it?

**Comment by ivernus** [comment]
> @ivernus yes, it looks very good. A couple of files are missing (e.g. help.txt translation),
> but it is perfect finally have the Irish translation even without the help file.
> Could you please make a pull-request and I will merge and deploy it?

No worries, there is PR #456 . I will try to do the help file also and raise a new PR.

**Comment by chubin** [comment]
@ivernus Conor, the translation is activated:

https://ga.wttr.in

I added your name to the translators list:

https://wttr.in/:translation

Please say if the name should be removed/hidden.

Could you please also translate the v2 view, it is just a bunch of words:

https://github.com/chubin/wttr.in/blob/master/lib/translations_v2.py#L29

**Comment by chubin** [comment]
(I've added *todo* to the header)
